addons/account/wizard/account_invoice_refund.py

```diff
@@ -59,6 +59,15 @@ def compute_refund(self, mode='refund'):
                 description = form.description or inv.name
                 refund = inv.refund(form.date_invoice, date, description, inv.journal_id.id)
 
+                for refund_invoice in refund.invoice_line_ids:
+                    proerty_income_id = refund_invoice.product_id.property_account_income_id.id
```

Comment:
property spelling.

but why do this like this anyway.  that is the point of _get_product_accounts method.